src/SFA.DAS.AANHub.Application/SFA.DAS.AANHub.Application.csproj

```diff
@@ -21,6 +21,7 @@
   <ItemGroup>
     <Folder Include="Apprentices\Commands\" />
     <Folder Include="Common\Validators\RequestedByMemberId\" />
+    <Folder Include="Employers\Queries\" />
```

Comment:
This is definitely not needed, perhaps it got added when the folder was created with no files.